addons/skin.estuary/xml/Includes_PVR.xml

```diff
@@ -246,23 +246,31 @@
 				<width>830</width>
 				<height>262</height>
 				<font>font36_title</font>
-				<label>$INFO[ListItem.Title]</label>
+				<label>$INFO[ListItem.Title] $INFO[ListItem.Year,([COLOR grey],[/COLOR])]</label>
 				<visible>!PVR.HasEpg</visible>
 			</control>
 			<control type="label">
 				<top>390</top>
 				<width>830</width>
 				<height>262</height>
 				<font>font36_title</font>
-				<label>$INFO[ListItem.EpgEventTitle]</label>
+				<label>$INFO[ListItem.EpgEventTitle] $INFO[ListItem.Year,([COLOR grey],[/COLOR])]</label>
 				<visible>PVR.HasEpg</visible>
 			</control>
 			<control type="label">
 				<top>435</top>
-				<width>830</width>
+				<width>40%</width>
```

Comment:
Now, there is far less space for the episode title, which can be pretty long. :-/ 

Have you tested this with long episode titles?

I'm afraid, what works fine for the guide window because there is enough space, will not be a good layout for the info dialogs.

Reply:
Oops, this is where my rudimentary skins knowledges bites me. Ok, will move it above like your original suggestion.

Reply:
So shortening the following line changes the width of the line above? Let me test this first, it may still be ok.

Reply:
Be careful, when adding genre "above", a progress bar position needs to be adjusted as well.

=> https://github.com/phunkyfish/xbmc/blob/80e1a678343b409c98ed90d1c2ebb8eeecd8b9f2/addons/skin.estuary/xml/Includes_PVR.xml#L218

Most probably the height of the label above the progress needs also adjustment

=> https://github.com/phunkyfish/xbmc/blob/80e1a678343b409c98ed90d1c2ebb8eeecd8b9f2/addons/skin.estuary/xml/Includes_PVR.xml#L214

Best test scenario is to check how it looks if all data (title, label with more data (count is not static!), progress, ..) is present.

Reply:
Or is it episode name you meant instead of title?

Reply:
Before:

![screenshot000](https://user-images.githubusercontent.com/3226626/66297389-e0ed8100-e8ef-11e9-89bd-79c49218f178.png)

After:

![screenshot001](https://user-images.githubusercontent.com/3226626/66297407-e8ad2580-e8ef-11e9-887f-0788b39a682f.png)

TBH, I don't like it.

Reply:
Fair, it needs to be changed.